spec/order_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'order'
+
+RSpec.describe Order do
+  
+  before :each do
```

Comment:
This is great, in that it works, so well done!  It would be more idiomatic to use `let`, rather than an instance var, however.